api_code/db/1_db_init.py

```diff
@@ -0,0 +1,15 @@
+'''

```

Comment:
This entire file may not be needed since this functionality is performed when inserting into the table using the `db_utils.py`, but it is good to have anyways.